### PR TITLE
Wgoulet/example gw

### PR DESCRIPTION
**Pull request by waltergoulet-okta**
Adds an example for creating a Gateway, adds/updates Readme.md in a few places.

### CODE REVIEW

File: examples/oktapam_gw_setup/gw_setup_example.tf [hunk 1]
```diff
@@ -0,0 +1,69 @@
+terraform {
```
Comment (sachinsaxena-okta) [inline examples/oktapam_gw_setup/gw_setup_example.tf:1]: nit You can separate the provider config part in a file main.tf. For oktapam provider version, we can just put >= 0.1.4.

File: examples/oktapam_gw_setup/gw_setup_example.tf [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    zone	= "us-central1-a"
+}
+
+# Create gateway setup token
+resource "oktapam_gateway_setup_token" "test-gw" {
+  description          = "tf-gw"
+  labels               = {env:"prod"}
+}
+
+output "gw_id" {
```
Comment (sachinsaxena-okta) [inline examples/oktapam_gw_setup/gw_setup_example.tf:29]: We can move this to separate file outputs.tf.

**Comment by sachinsaxena-okta** [comment]
Check doc generation failure too. Again generate the docs using - make generate

File: examples/oktapam_gw_setup/gw_setup_example.tf (hunk 1 above)
Comment (waltergoulet-okta) [reply examples/oktapam_gw_setup/gw_setup_example.tf:1]: Done

File: examples/oktapam_gw_setup/gw_setup_example.tf (hunk 2 above)
Comment (waltergoulet-okta) [reply examples/oktapam_gw_setup/gw_setup_example.tf:29]: Done thanks

**Comment by sachinsaxena-okta** [comment]
Did you commit main.tf and outputs.tf files ?

**Review by sachinsaxena-okta — APPROVED** [review]

**Pull request merged**